app/src/main/java/com/SteadyView/SteadyView/MainActivity.java

```diff
@@ -306,9 +306,14 @@ protected void onActivityResult(int requestCode, int resultCode, Intent data){
         }
         System.out.println("spokenText: " + spokenText);
         if(spokenText != "") {
-            String temp = urlizeText(spokenText);
-            String url = prependHTTP(temp);
-            web.loadUrl(url);
+            if (spokenText.substring(0, 7).equals("search ")) {
```

Comment:
Does changing this to `if (spokenText.length() > 7 && spokenText.substring(0, 7).equals("search ")) {` prevent this from getting an error? @Colecf